packages/storybook/src/theme.ts

```diff
@@ -1,5 +1,79 @@
+const breakpoints = ['640px', '768px', '1024px', '1280px']
+
+const baseColors = {
```

Comment:
💯 like the split out and being able to refer to the object instead of the just the string

Reply:
Yea, agreed. Glad you pointed out the Tailwind theme.